cfgov/paying_for_college/disclosures/scripts/load_programs.py

```diff
@@ -9,19 +9,25 @@
 
 
 NO_DATA_ENTRIES_LOWER = ("", "blank", "no grads", "no data", "none")
+VALIDATED_S3_DATA_URL = (
+    "https://files.consumerfinance.gov"
```

Comment:
Would we want/be able to use the `AWS_S3_CUSTOM_DOMAIN` setting here?

Reply:
That's where I'm not sure legacy would be affected. This sounds right, but until Akamai work is complete and tested, I think this is the safe conversion step.

Reply:
Nah, I think you're right. Thanks for pointing that out!